tiled/server/app.py

```diff
@@ -298,8 +300,23 @@ async def unsupported_query_type_exception_handler(
         allow_credentials=True,
         allow_methods=["*"],
         allow_headers=["*"],
+        # allow_headers=["X-Requested-With", "X-Request-ID"],
```

Comment:
A glimpse of what's to come?

Reply:
Yes. I need to look into the implications of locking this down.

Reply:
…and this is a note to remember to let these through.

Reply:
I elaborated as such in the comment in a new comit.